velodrome/README.md

```diff
@@ -104,3 +104,7 @@ New project deployments
 -----------------------
 
 - Create [secret for InfluxDB](grafana-stack/#first-time-only)
+- Deploy everything: `./config.py | kubectl apply -f -l project=${NEW_PROJECT_NAME}`
+- Once the service has the IP, connect to the grafana instance, and add the
```

Comment:
Yes, once the service is deployed for the first time, you need to register the IP (from `kubectl describe svc`) as static in gcp, and then add it to the config file. If you want a static ip of course.